app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -77,6 +71,24 @@ class HomeViewModel @Inject constructor(
         
     """.trimIndent()
 
+    // TODO(localization): localize if needed
+    fun giveFeedbackEmailTemplate(deviceHash: String? = "unavailable"): String = """
```

Comment:
So you would like me to create a class called `EmailComposer`? Why not EmailViewModel or something like that?

Where would I place the class?

Reply:
I think, for clarity, the name "xxxViewModel" should have those that actually extend the `ViewModel` and are closely related to the fullsize screens. Such `EmailComposer` could be reused in multiple ViewModels. It could be placed just in `com.wire.android.util`, where there is a `CustomTabsHelper` for instance. 😉